stream_alert_cli/runner.py

```diff
@@ -84,6 +88,25 @@ def lambda_handler(options):
         stream_alert_test(options)
 
 
+def configure_handler(options):
+    """Configure StreamAlert main settings
+
+    Args:
+        options [named_tuple]: ArgParse command result
+    """
+    if options.config_key == 'prefix':
+        if type(options.config_key) not in (unicode, str):
+            LOGGER_CLI.error('Invalid prefix type, must be string')
+            return
+        CONFIG.set_prefix(options.config_value)
+
+    elif options.config_key == 'aws_account_id':
+        if not re.search(r'\A\d{12}\Z', options.config_value):
```

Comment:
Since regex is inherently slow, can't we just check for a len of 12 and str.isdigit()?